deny.toml

```diff
@@ -52,7 +52,7 @@ skip = [
   { name = "libloading" },     # wgpu-hal itself depends on 0.8 while some of its dependencies, like ash and d3d12, depend on 0.7
   { name = "memoffset" },      # tiny dependency
   { name = "quick-xml" },      # old version via wayland-scanner
-  { name = "redox_syscall" },  # old version via directories-next
+  { name = "redox_syscall" },  # old version via winit
```

Comment:
I'm not exactly sure what is going on in this file, but with this change, `winit` is the crate referring to the oldest version of `redox_syscall` (0.3.5)